highfidelity/hifi_tests
# Use fixed orientation in lieu of the avatar's orientation.

# Description
Images have been updated
# Testing
1. Run https://github.com/NissimHadar/hifi_tests/blob/fixZoneEffectsTest/tests/content/entity/zone/zoneEffects/test.md
Images should be as in folder
2. Run https://github.com/NissimHadar/hifi_tests/blob/fixZoneEffectsTest/tests/testRecursive.js
Images should be as in folder
